src/content/reference/manifest.mdx

```diff
@@ -865,6 +865,17 @@ If the url value is declared in both places, the one declared in the deploy sect
 
 A relative path to a markdown file. The contents of the file will be displayed in the dashboard. You can use this to describe the resource or share instructions on how to use it.
 
+[GitHub Flavored Markdown (GFM)](https://github.github.com/gfm/) is supported, which adds the following to standard [Markdown](https://www.markdownguide.org/):
+- Autolink literals (`www.x.com`)
+- Footnotes (`[^1]`)
+- Strikethrough (`~~stuff~~`)
+- Tables (`| cell |…`)
+- Tasklists (`* [x]`)
+
+Images are allowed, as long as they referenced either by `data:` or `https://`. Images from unsupported protocol sources or relative paths will not render and will be replaced with a warning message pointing to this document.
```

Comment:
> Images are allowed, as long as they **are** referenced either by `data:` or `https://`. Images from unsupported protocol sources or relative paths will not render and will be replaced with a warning message pointing to this document.

Minor edit, but otherwise this looks good to me 👍

Reply:
Thanks @omnomagonz! Good catch. I've updated in my last commit, preview should be updated in a second.